### ISSUE TITLE
Controlling different fans through the same fan header, RPM jumping randomly

### ISSUE THREAD

**Issue by durrburger93** [Rem0o/FanControl.Releases#2535]
**Describe the bug**
I'm seeing weird behavior when trying to use different types of fans on the same fan header, and control them with Fan Control. When using multiple fans of the same model on the same header, there are no issues and they work normally, but now I'm trying to control 2 a12x25s and 1 Phanteks t30 through one header, as they are all zip-tied to my 4080. T30 is set to its "performance" mode at 2000 rpm, so all 3 fans have the same rpm within a margin of error. 

When doing this and setting up any curve or manual control to a fixed speed, the listed rpm jumps wildly at every setting. For example, when setting the header to 50%, I see card speeds jump from 850/980/2000/1150 rpm etc, it's just showing different numbers and I'm not sure if the fans are constantly getting different signals (possibly damaging them with constant changes if that's possible?), or if the program is just reporting random speeds.

I'm also seeing the question mark indicating that the reports are inaccurate and that something may be overriding the settings, and I'm not using any other software. All the fans are set to 0 rpm at every temp in BIOS, and this works normally for all the other fans using splitters as long as they are the same fan model. 

So TLDR, using different types of fans on the same header is reporting/sending inaccurate signals to the fans.

**Comment by Rem0o** [comment]
How are said fans connected to the same header? Might the rpm be mixed on all 3?  Such wild RPM swings, you would definitely hear it.  

If RPM values for that header are unreliable, delete any calibration on that control.

**Comment by durrburger93** [comment]
Here is how the fans are connected basically, and the T30 goes into the main 4 pin port of the Y splitter so it should in theory control all 3 fans:
![fans](https://github.com/Rem0o/FanControl.Releases/assets/82004352/63deadf1-2853-4aae-a082-581af7ea38b1)

I did try calibration paired controls automatically but it didn't help as it was doing the same thing afterward. Should I remove the paired sensor or do something else?
